### ISSUE TITLE
Trailing slash on export_root

### ISSUE THREAD

**Issue by bschlaman** [google/exposure-notifications-server#1135]
### TL;DR
Exportimporter does not add a "/" in between export_root and export filepath if one is missing when importing the exportfiles.

**Expected behavior**
Exportimporter should check if a "/" is included in the paths listed in an index.txt file, and if one is not there, add it on to the export_root before importing exports.

**Observed behavior**
Exportimporter will append whatever is in the index.txt file onto the xport_root, and will miss a "/" if its is not present either at the beginning of rows in index.txt or at the end of export_root.

### Reproduction
Create an exportimport config with no trailing "/" and import exports that do not begin with a "/"
